content/en/docs/ops/common-problems/validation/index.md

```diff
@@ -17,171 +17,121 @@ necessary.
-    service:
-      # service corresponds to the Kubernetes service that implements the
-      # webhook, e.g. istio-galley.istio-system.svc:443
-      name: istio-galley
-      namespace: istio-system
-      path: /admitmixer
-  failurePolicy: Fail
-  name: mixer.validation.istio.io
-  namespaceSelector: {}
-  rules:
-  - apiGroups:
-    - config.istio.io
-    apiVersions:
-    - v1alpha2
-    operations:
-    - CREATE
-    - UPDATE
-    resources:
-    - rules
-    - attributemanifests
-    - circonuses
-    - deniers
-    - fluentds
-    - kubernetesenvs
-    - listcheckers
-    - memquotas
-    - noops
-    - opas
-    - prometheuses
-    - rbacs
-    - servicecontrols
-    - solarwindses
-    - stackdrivers
-    - statsds
-    - stdios
-    - apikeys
-    - authorizations
-    - checknothings
-    - listentries
-    - logentries
-    - metrics
-    - quotas
-    - reportnothings
-    - servicecontrolreports
-    - tracespans
+    - '*'
+    scope: '*'
+  sideEffects: None
+  timeoutSeconds: 30
 {{< /text >}}
 
 If the `validatingwebhookconfiguration` doesn’t exist, verify the
-`istio-galley-configuration configmap` exists. `istio-galley` uses
+`istio-validation` `configmap` exists. Istio uses
 the data from this configmap to create and update the
 `validatingwebhookconfiguration`.
 
 {{< text bash yaml >}}
-$ kubectl -n istio-system get configmap istio-galley-configuration -o jsonpath='{.data}'
-map[validatingwebhookconfiguration.yaml:apiVersion: admissionregistration.k8s.io/v1beta1
+$ kubectl -n istio-system get configmap istio-validation` `-o jsonpath='{.data}'
```

Comment:
Looks like there are unintended backtick characters in this command, rendering it incorrect.

Reply:
fixed ( Ithink)